app/src/store/voice/call/callInvite.ts

```diff
@@ -0,0 +1,276 @@
+import {
+  createEntityAdapter,
+  createSlice,
+  miniSerializeError,
+  type PayloadAction,
+  type SerializedError,
+} from '@reduxjs/toolkit';
+import { type CallInvite as TwilioCallInvite } from '@twilio/voice-react-native-sdk';
+import { match } from 'ts-pattern';
+import {
+  getCallInfo,
+  getCallInviteInfo,
+  type CallInfo,
+  type CallInviteInfo,
+} from './';
+import { type AsyncStoreSlice } from '../../app';
+import { createTypedAsyncThunk, generateThunkActionTypes } from '../../common';
+import { callInviteMap } from '../../../util/voice';
+import { settlePromise } from '../../../util/settlePromise';
+
+const sliceName = 'callInvite' as const;
+
+/**
+ * Receive call invite action.
+ */
+export const receiveActionTypes = generateThunkActionTypes(
+  `${sliceName}/receive`,
```

Comment:
why interpolate, and not just use `callInvite`? It doesn't seem like `sliceName` is ever changed from `callInvite`

Reply:
I'd like to eliminate as many "magic words" as possible and just make a string once, and reuse it everywhere.